new/LIT6911MalkeaAlam.xml

```diff
@@ -51,7 +56,21 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     </teiHeader>
     <text>
         <body>
-            <div type="bibliography"/><!---->
+            <div type="bibliography"/>
+
+            <div type="edition">
+            <div type="textpart" subtype="incipit">
+              <ab>
+                <lg>
```

Comment:
would it also be possible to add an explicit?

Reply:
Added, but with `<space>` for the lacuna at the start of the line which we are still discussing how best to fill from an editor's standpoint.

Reply:
Scratch that, `<space>` didn't validate there, but it's not a huge concern. We can add the proper encoding for the editorial suggestion once it goes to print (whenever and wherever that will be).